Userland/Libraries/LibWeb/Page/Page.h

```diff
@@ -315,7 +315,7 @@ class PageClient : public JS::Cell {
 
     virtual void page_did_change_audio_play_state(HTML::AudioPlayState) { }
 
-    virtual WebView::SocketPair request_worker_agent() { return { -1, -1 }; }
+    virtual WebView::SocketPair request_worker_agent() { return { IPC::File::adopt_fd(-1), IPC::File::adopt_fd(-1) }; }
```

Comment:
Instead of all the `IPC::File::adopt_fd(-1)` in this patch, we could just do `IPC::File {}` to default construct it. A bit less noise, and hides the magic `-1` value

Reply:
good call, I'll push that up